File: catsAndMouse.py
n = int(input())
arr = list()
l = list()
for i in range(0, int(n)):
  x = input().split(" ")
  arr.append(x)

for i in range(0, n):
    for j in range(0, 3):
        arr[i][j] = int(arr[i][j]);

for i in range(0 , n):
    cata = arr[i][0];
    catb = arr[i][1];
    mouse = arr[i][2];
    dis1 = mouse - cata;
    dis2 = catb - mouse;
    if dis1 < 0:
        dis1 = -1 * dis1
    if dis2 < 0:
        dis2 = -1 * dis2
    if dis1 < dis2:
        print("Cat A")
    elif dis2 < dis1:
        print("Cat B")
    else:
        print("Mouse C")

File: catsAndMouse.java
import java.util.*;
import java.io.*;

public class catsAndMouse{

  // static void findResult(List<Integer> a){
  //   int cata = a.get(0);
  //   int catb = a.get(1);
  //   int mouse = a.get(2);
  //   if(catb - mouse == mouse - cata){
  //     System.out.println("mouse C");
  //   }
  //   if(catb - mouse > mouse - cata){
  //     System.out.println("cat B");
  //   }
  //   else if(catb - mouse < mouse - cata){
  //     System.out.println("cat A");
  //   }
  // }

  public static void main(String args[]) throws IOException{
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    int n = Integer.parseInt(br.readLine());
    List<Integer> li = new ArrayList<Integer>();
    for(int i = 0; i<n; i ++){
      String lines = br.readLine();
      String strs[] = lines.trim().split("");
      li.get(i) = strs[i];
    }

    // for(int )
    System.out.println(li.get(0));
  }
}
